docs/reference/plugins.rst

```diff
@@ -182,6 +182,16 @@ Can the plugin be inserted inside the text plugin?  If this is ``True`` then
 
 See also: `icon_src`_, `icon_alt`_
 
+.. _system_plugin:
+
+system
+------
+
+Default: ``False``
+
+Plugins with this attribute set to True can be dropped in any placeholder, overriding
+any placeholder configuration in :setting:`CMS_PLACEHOLDER_CONF`.
```

Comment:
do we want this to be a public API? The naming of it alone sounds like something internal that shouldn't be public (possibly even not using that attribute but a hardcoded list of whitelisted plugins). I'm not necessarily saying it shouldn't be a public API, just wondering.